marklogic-data-hub/src/test/java/com/marklogic/hub/flow/TemporalWriteTest.java

```diff
@@ -75,15 +79,45 @@ void writeTemporalFile() {
             "declareUpdate();\n" +
             "temporal.documentDelete('temporal/test', '/temporal/ingestion/test.json');";
 
-        try{
+        try {
             runAsDataHubOperator().newStagingClient().newServerEval().javascript(deleteTemporalDoc).eval();
-        }
-        catch (Exception e){
+        } catch (Exception e) {
             logger.error("Document deletion failed: ", e);
             Assertions.fail("dh-operator should be able to delete the document");
         }
     }
 
+    @Test
+    void ingestDocWithTemporalCollecion() {
+        String collections = "fruits,stuff,temporal/test";
+        ObjectNode workUnit = objectMapper.createObjectNode();
+        workUnit.put("uriprefix", "/bulkJavaTest/");
+        workUnit.put("collections", collections);
+        runAsDataHubOperator();
+        InputEndpoint endpoint = InputEndpoint.on(
+            getHubClient().getStagingClient(),
+            getHubClient().getModulesClient().newTextDocumentManager().read("/data-hub/5/data-services/ingestion/bulkIngester.api", new StringHandle())
```

Comment:
Pinging @anu3990 - let's get this PR merged before your PR is merged that moves this module, that will avoid merge conflicts.